app/jobs/make_phone_calls_job.rb

```diff
@@ -0,0 +1,28 @@
+class MakePhoneCallsJob < ApplicationJob
+  queue_as :phone_calls
+
+  require 'net/http'
+  require 'open-uri'
+
+  def perform(args)
+    # Find all the phone stores for which there an activecalendar for the current day of the week
+    # For each store, create a job to make the request
+
+    Rails.logger.debug("Going to make a phone call to #{args[:phone]}")
+
+    phone = args[:phone]
+    store = args[:store] || '12345' # random number in case there's no store
+    endpoint = ENV['APIGEE_URL'].gsub('_store_id_', store)
```

Comment:
Can you add sample env variables to .env.sample? I'd like to see the URL format, to see if it would be appropriate to use rfc6570 through the addressable gem.

Reply:
I will